.circleci/config.yml

```diff
@@ -823,8 +823,6 @@ jobs:
 
   backend-integration-tests-offline:
     <<: *base-job
-    # These tests are even flakier running on M1
-    resource_class: macos.x86.medium.gen2
```

Comment:
Just to check but should we make sure it works before merging? (maybe you have already). We can just add it to the workflow executed on every commit temporarily, make sure they pass, then revert that commit.

Reply:
I'll check and verify!

Reply:
@tonidero The job passed! I'll revert the changes I made to run it in the CI here and then merge it into main.

Reply:
Perfect! Thanks! 🎉